t/t.go

```diff
@@ -206,6 +211,9 @@ func runTestsFor(ctx context.Context, pkg, prefix string) error {
 	if isTeamcity {
 		args = append(args, "-json")
 	}
+	if *runCov {
+		args = append(args, fmt.Sprintf("-covermode=%s", testCovMode), fmt.Sprintf("-coverprofile=%s", tmpCovFile))
```

Comment:
@kevinmingtarja - this will work only when we are running the tests in serial mode. This will fail when we parallelize the tests.

I think we should talk about this tomorrow, but I won't block this merge. My main reason why this should still go in, is as follows: we know for sure that our test-harness has not been stabilized for parallelization. So I will be adding a comment here in your PR, to fix this later.